src/app/routes/liveRadio/getInitialData/index.js

```diff
@@ -19,6 +19,7 @@ export default async ({ path: pathname }) => {
   const liveRadioDataPath = overrideRendererOnTest(pathname);
   const { json, ...rest } = await fetchPageData(liveRadioDataPath);
   const contentData = path(['content'], json);
+  const pageType = { metadata: { type: 'Live Radio' } };
```

Comment:
> This fix is to use an alternative variable - pageData.contentType

Why don't you use contentType that is already passed through in pageData rather than adding back metadata?

Reply:
I did implement that initially https://github.com/bbc/simorgh/pull/6389/commits/8d1a505c2507f12529fc580f0d9dd7648c4297ed - but then I saw that a similar thing was done in another PR - https://github.com/bbc/simorgh/pull/6385, so kept it consistent with that. Now I am torn, because can't decide which solution is better!

Reply:
Have decided to go back to the original implementation, thanks!

Reply:
Ah I see. I actually prefer your newer implementation as it is simpler to read. Once everyone refactors to only pass what is necessary, we can remove 'metadata'. Sorry if you've reverted between messages! I wanted to understand the switch from `contentType` to `type`

Reply:
Ah shit I see I was too slow haha

Reply:
Hahahha 🙈 I'll put it back to the newer implementation, and we can keep an eye on it when it comes to doing Radio Brand & TV pages